index.html

```diff
@@ -45,7 +45,7 @@
           <a class="dv-logo" href="index.html">
             <i class="fa fa-database"></i>
             <span>Dataverse Global<br /></span>
-            <span>Submit 2023</span>
+            <span>Summit 2023</span>
           </a>
           <ul class="nav-list">
             <li><a class="about" href="about.html">About</a></li>
```

Comment:
- [ ] From the project requirements, you should stick to [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), But as you can see, the style (background-color, width, height, and font) of the text `A tech-event designed to ..` does not match [Cindy's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Also, the `@ AUCC, Addis Ababa, Ethiopia` margin left should be the same as the `2023.03.20(MON) ~ 23(WEN)`, and make sure it takes one line.
- [ ] Make sure that you adjust the margins to match the Design :blush: 

| Your Design |Cindy Design |
| :---:   | :---: | 
| ![image](https://user-images.githubusercontent.com/92303891/227339885-41c1759f-4be1-4a07-a88d-d1711569f67d.png)| ![image](https://user-images.githubusercontent.com/92303891/227340035-956c82a8-a275-487e-b83a-8b048bacc8ad.png)|

Reply:
Done